fsdb/Fsdb.py

```diff
@@ -49,7 +56,7 @@ def __init__(self, fsdbRoot, mode=None, deep=None, hash_alg=None):
             raise Exception("fsdb can not operate on relative path")
 
         # on different platforms same unicode string could have different rappresentation
-        if isinstance(fsdbRoot, unicode):
+        if isinstance(fsdbRoot, str):
```

Comment:
yes, but on line 7 I do `from builtins import str` which will change the names to be the same as in python3.
Anyway, I think that the best solution is to add unit tests and see if this pull request will make them fail.